tests/tests_lite/utilities/test_imports.py

```diff
@@ -27,9 +27,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from lightning_lite.strategies.fairscale import _FAIRSCALE_AVAILABLE
```

Comment:
Okay. Do you know what the test_imports.py file is for?

Reply:
My comment above is incorrect. I thought this was something else. Sorry!

`test_imports` was added because we had an issue where `*_AVAILABLE` returned False, but the library could be imported (or was it vice versa?)

In any case, this test should be in either PL or Lite, but not both

Reply:
Got it. Following up on this by importing _FAIRSCALE_AVAILABLE in PL from Lite and then move the test accordingly.